package.json

```diff
@@ -17,11 +17,12 @@
     "dbmigratetest": "db-migrate --migrations-dir 'db/migrations/' -e test",
     "dbmigratedry": "db-migrate --dry-run --migrations-dir 'db/migrations/'",
     "cypress-run": "cypress run --browser firefox",
+    "cypress-open": "cypress open --browser firefox",
```

Comment:
Useful for debugging issues with UI tests, cypress-run will close cypress when it's done, whether successful or not.